specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2015-01-01/resourcehealth.json

```diff
@@ -548,13 +548,6 @@
       "description": "The name of the resource group.",
       "x-ms-parameter-location": "method"
     },
-    "ResourceTypeParameter": {
```

Comment:
it appears this wasn't been used in the spec or any others in the repo, right? so removing it shouldn't cause any harm.

Reply:
it was not being used. and having it in the global parameters section was causing this to be a client property.